src/processing/1_Verden/README.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Lærerveiledning - Verden
+level: 2
```

Comment:
vet ikke hvorfor denne ble merget, eg trykket ikke på det.... hadde tenkt å be om endring på nivå og så merge når det ble fikset. @steinoro : fikser du nivå og sender ny PR?